vm-executor/src/opcode_check_used.rs

```diff
@@ -0,0 +1,22 @@
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum OpcodeCheckUsed {
+    MemoryCopy,
+    MemoryFill,
+}
+
+impl OpcodeCheckUsed {
+    pub fn from_value(value: u32) -> Option<Self> {
+        match value {
+            0 => Some(Self::MemoryCopy),
```

Comment:
this is not good. like we would need the full set here. to be easily usable from spaceVM.